sdk/src/types/block/semantic/mod.rs

```diff
@@ -125,10 +129,36 @@ impl<'a> SemanticValidationContext<'a> {
             }
         }
 
+        // Validation of inputs.
+
+        let mut has_implicit_account_creation_address = false;
+
         for (index, (output_id, consumed_output)) in self.inputs.iter().enumerate() {
             let (amount, consumed_native_token, unlock_conditions) = match consumed_output {
                 Output::Basic(output) => (output.amount(), output.native_token(), output.unlock_conditions()),
-                Output::Account(output) => (output.amount(), None, output.unlock_conditions()),
+                Output::Account(output) => {
+                    if output.features().block_issuer().is_some() {
+                        let account_id = output.account_id_non_null(output_id);
+
+                        if self.commitment_context_input.is_none() {
```

Comment:
just checking ... so at this point any commitment context input will do to pass validation, no constraints?